site/content/contribute/plugins/_index.md

```diff
@@ -5,4 +5,8 @@ section: "contribute"
 weight: 5
 ---
 
-Mattermost plugins are isolated pieces of code, separate from the main repositories, used to extend the functionality of the Mattermost server and webapp. Please see the [Plugin Developer Documentation](/extend/plugins) to learn more. If you would like to contribute to plugin projects, please see the [Help Wanted Tickets](/contribute/plugins/help-wanted-tickets) tab.
+Mattermost plugins are isolated pieces of code, separate from the main repositories, used to extend the functionality of the Mattermost server and webapp. Plugins are written in Go and/or React. The Go portions run directly on the Mattermost server, and are managed by the server at runtime. The React portions run in each user's browser, allowing developers to modify the user interface in [several ways](https://developers.mattermost.com/extend/plugins/webapp/best-practices).
```

Comment:
```suggestion
Mattermost plugins are isolated pieces of code written in Go and/or React. They're separate from the main repositories and are used to extend the functionality of the Mattermost server and webapp. 

- The Go portions run directly on the Mattermost server, and are managed by the server at runtime. 
- The React portions run in each user's browser, allowing developers to modify the user interface in [several ways](https://developers.mattermost.com/extend/plugins/webapp/best-practices).
```

Reply:
Not sure whether the formatting will carry over to the actual content but easily remedied if not. Also, 0/5 on whether it should be bulleted or a separate paragraph.